### DISCUSSION TITLE
How are home-brew packages maintained?

### DISCUSSION THREAD

**Discussion by infa-jowoods** [Homebrew/discussions#1989] [Everyday usage] [answered]
Who maintains home-brew packages? It seems largely volunteers looking at the commits.

How is this done? How can a small number of folks stay on top of evolving software from around the globe?

**Comment by SMillerDev** [comment, marked as answer]
> Who maintains home-brew packages?

The community, anyone can make a pull request and update/add software. Maintainers are just there to provide guidance and consistency. And we have [a lot of CI](https://mikemcquaid.com/2021/06/09/robot-pedantry-human-empathy/) to make sure that the humans have to do as little repetitive work as possible.

> How can a small number of folks stay on top of evolving software from around the globe?

Sometimes I wonder that myself.

**Reply by infa-jowoods** [reply]
Cool :) is it the same model for apt on Linux?

**Reply by SMillerDev** [reply]
You'd have to ask the Debian developers that, but judging by the Debian mailing lists they have specific maintainers for package groups: https://lists.debian.org/completeindex.html. So that would mean that the updates and additions are done by specific people responsible for only a subset of packages.
